src/OpenLoco/src/Map/StationElement.h

```diff
@@ -57,12 +58,21 @@ namespace OpenLoco::World
             _stationId &= ~0x3FF;
             _stationId |= enumValue(id) & 0x3FF;
         }
-        void setUnk7SLR2(uint8_t val)
+        // ((_6 & 0xFC00) >> 10) Note: Only none 0 for airports
```

Comment:
By `none 0`, do you mean `non-zero`? Just making sure.

Reply:
Yes sorry in my head it made sense